Fix redirection for root (#34)
wireapp/wire-docs: src/how-to/administrate/elasticsearch.rst

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

See also the elasticsearch sections in :ref:`investigative_tasks`.

Troubleshooting
~~~~~~~~~~~~~~~

Description:
**ES nodes ran out of disk space** and error message says: ``"blocked by: [FORBIDDEN/12/index read-only / allow delete (api)];"``

Solution:

> * clean up disk (e.g. ``apt autoremove`` on all nodes) & restart nodes a/o ES process

Reviewer: ```suggestion
* clean up disk (e.g. ``apt autoremove`` on all nodes), then restart machines and/or the elasticsearch process
```

> * get ES out of *read-only* mode: ``curl -X PUT -H 'Content-Type: application/json' https://${ES_HOST}:${ES_PORT}/_all/_settings -d '{"index.blocks.read_only_allow_delete": null}'``

Reviewer: For consistency with the rest of the page, as "ES_HOST/ES_PORT" are not defined.

```suggestion
* get the elastichsearch cluster out of *read-only* mode: SSH to one elasticsearch machine, then run ``curl -X PUT -H 'Content-Type: application/json' http://localhost:9200/_all/_settings -d '{"index.blocks.read_only_allow_delete": null}'``
```

> * trigger reindexing: ``curl -v -X POST $BRIG_IP:8080/i/index/reindex``

Reviewer: ```suggestion
* trigger reindexing: From a kubernetes machine, one one terminal port-forward to into brig useing ``kubectl port-forward svc/brig 9999:8080``, and in a second terminal trigger the reindex: ``curl -v -X POST localhost:9999/i/index/reindex``
```

Reviewer 2: > *...port-forward to into brig...*

Not sure, but it feels a bit hard to read. maybe *to* is unnecessary here?

Reviewer: Ah sorry, multiple typos in my sentence, I updated the suggestion.
